Dockerfile

```diff
@@ -1,8 +1,9 @@
-ARG XX_VERSION=1.0.0-rc.2
+ARG GO_VERSION=1.17.5
```

Comment:
Can we please not hardcode the Go patch version

Reply:
Yes, I'm going to make this change everywhere I also just noticed it and am preparing a PR for NC and SC.